roles/ovirt-datacenter-cleanup/README.md

```diff
@@ -17,6 +26,9 @@ Role Variables
 |--------------------------|-----------------------|--------------------------------------|
 | data_center_name         | UNDEF                 | Name of the data center.             |
 | format_storages          | false                 | Whether role should format storages when removing them. |
+| hosted_engine            | false                 | Specify whether we datacenter to cleanup contains a VM which is serving an OVirt hosted engine installation.|
```

Comment:
because we know that hosted engine vm name(HostedEngine) and hosted engine storage domain name(hosted_storage) are constants.
maybe it will be better to detect it dynamically...
and remove this  hosted_engine parameter